src/components/Home.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import BookList from './BookList';
+import BookForm from './BookForm';
+
+const Home = () => {
+  const [books, setBooks] = useState([]);
+
+  const handleCreateBook = (title) => {
+    const newBook = { id: Date.now(), title };
+    setBooks((prevBooks) => [...prevBooks, newBook]);
+  };
+
+  const handleDeleteBook = (id) => {
+    setBooks((prevBooks) => prevBooks.filter((book) => book.id !== id));
+  };
```

Comment:
- [ ] Well, to ensure consistency with the project requirements, I suggest implementing a similar approach when deleting a book. You should utilize the `deleteBook` reducer and dispatch it to remove the designated book from the Redux store. This approach aligns with the project's specified guidelines for data manipulation. Your dedication to adhering to the project's directives is commendable, and I encourage you to update your code accordingly.

Reply:
- Refactor all files to use Redux in React components to solve bugs.